code-samples/popular-topics/canvas/13/index.js

```diff
@@ -44,10 +48,12 @@ client.on('interactionCreate', async interaction => {
 		context.closePath();
 		context.clip();
 
-		const avatar = await Canvas.loadImage(interaction.user.displayAvatarURL({ format: 'jpg' }));
+		const avatarRes = await fetch(interaction.user.displayAvatarURL({ format: 'jpg' }));
```

Comment:
As I had said in #1036, `undici#fetch` is experimental, so I wouldn't recommend it. But by all means go ahead.

Reply:
Doesn't really mean much. The experimental label was added way before basic spec features were added such as fetching data or blob urls. Nowadays the implementation is mostly complete, minus some browser-only things (origin, CORs, etc.). Plus, the example follows the spec and with the vast amount of tests undici's fetch has, this example will never break

Reply:
(resolved)